api/app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Exceptions;
+
+use Illuminate\Support\MessageBag;
+use Illuminate\Http\Exception\HttpResponseException;
+
+class ValidationException extends HttpResponseException
```

Comment:
@xiphiaz I made a small tweak in dd31da5 to not hardcode it against ```ValidationException``` but to allow any custom exception class to override the response if needing to add more keys.